requirements/edx/base.txt

```diff
@@ -35,9 +35,10 @@ git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc432894737119
 django==1.8.9
 djangorestframework-jwt==1.7.2
 edx-django-oauth2-provider==0.5.0
+edx-djangorestframework-oauth==1.0.1
```

Comment:
The only thing the edx fork of djangorestframework-oauth is doing is modifying the requirements and setup.py. The project has an open PR to remove the django-oauth2-provider requirements from the runtime environment: https://github.com/jpadilla/django-rest-framework-oauth/pull/8/files.

I would be cool with dumping this fork if we could get that PR pushed upstream to remove that dependency.  I agree with you @rlucioni having to order these dependencies and shadow install the correct versions is brittle and I would prefer we do not adopt this habit.